nexus/db-queries/src/db/datastore/inventory.rs

```diff
@@ -639,6 +659,20 @@ impl DataStore {
                 }
             }
 
+            // Insert rows for all the physical disks we found.
+            {
+                use db::schema::inv_physical_disk::dsl;
+
+                let mut iter =
+                    disks.chunks(SQL_BATCH_SIZE.get().try_into().unwrap());
```

Comment:
In general I like the idea of inserting in batches.  But this is all going into one transaction so I'm not sure it's any better in that case.

Reply:
The batch size is still fairly large 1,000, so it's unlikely we'll be looping much regardless on a single rack or two. But I could see this blowing past the size of a "single SQL statement" if we consider it for larger multi-rack cases?